src/components/Calculator.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Button from "./Button";
+import ButtonsGroup from "./Buttons-group";
+export default function Calculator(props) {
+	return (
+		<div class="columns">
+			<div class="column is-one-fifth is-offset-5">
+				<div id={props.id} class="box">
+					<div class="container mb-2">
+
+						<input type="text" placeholder="0" id="calculator-screen"
+							class="input is-white has-text-right is-size-3 px-0 py-6 mb-4 is-shadowless has-text-weight-semibold"
+							size="8" />
+						<ButtonsGroup
+							groupsId="buttonsFirstLine" htmlClass="columns is-justify-content-center">
+
+							<Button htmlClass="is-large is-light m-2 is-fullwidth is-one-fifth has-text-danger" data="C">
+								<i class='icon fa-solid fa-c'></i>
```

Comment:
Where are the icons components?

Reply:
i did'nt create yeat, because i wanted to import them from a package
that's the reason i want to create another pr, to don't this pr that is for react with the icons stuff

Reply:
i think i'm gonna create the icons component normally, after merge this pr i'm gonna past them to react and install the packages

Reply:
done